src/Ast/BaseNode.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\PhpDocParser\Ast;
+
+abstract class BaseNode implements Node
+{
+
+	/** @var array<string, mixed> */
+	private $attributes = [];
+
+	/**
+	 * @param mixed $value
+	 */
+	public function setAttribute(string $key, $value): void
```

Comment:
I don't get this one. What's wrong here?

```bash
FILE: ...ome/runner/work/phpdoc-parser/phpdoc-parser/src/Ast/BaseNode.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 14 | ERROR | Unknown type hint "string" found for $value
    |       | (Squiz.Commenting.FunctionComment.InvalidTypeHint)
----------------------------------------------------------------------
```

Reply:
Probably a bug in an obsolete PHPCS version. I'm gonna look into this once I'm happy with the code and prepare to merge it 👍

Reply:
See https://github.com/phpstan/phpdoc-parser/pull/58 first

Reply:
#58 merged, please rebase this PR so we can see it passes :)